fizzbuzz/fizzbuzz.c

```diff
@@ -8,16 +8,42 @@
     every time that nothing gets printed and return the counter.
     Don't forget to include newlines '\n' in your printf statements!
 */
+
+// Run in terminal by these commands:
+// gcc -o fizzbuzz fizzbuzz.c
+// ./fizzbuzz
 int fizzbuzz(int n)
 {
-
+  int count = 0;
+  int i;
+  for (i = 0; i < n; i++)
+  {
+    if ((i % 5 == 0) && (i % 3 == 0))
```

Comment:
Great job putting this conditional first! You can also check `if (i % 15 == 0)`.